domain_director/blueprints.py

```diff
@@ -46,3 +50,30 @@ def list_nodes():
 @bp.route('/nodes.json', methods=['GET'])
 def list_nodes_json():
     return jsonify(Node.get_nodes_grouped())
+
+@bp_admin.route('/mesh/<int:mesh_id>/', methods=['PATCH'])
+def update_mesh(mesh_id):
+    try:
+        mesh_db_entry = Mesh.select().where(Mesh.id == int(mesh_id)).get()
+    except DoesNotExist:
+        abort(404)
+
+    if not 'switch_time' in request.values:
+        abort(400)
+
+    try:
+        switch_time = int(request.values['switch_time'])
+        switch_time_parsed = datetime.utcfromtimestamp(switch_time)
+    except ValueError:
+        return 'invalid switch time specified', 400
+
+    now = datetime.utcnow()
+
+    force = 'force' in request.values and request.values['force'].lower() in ['true', 'yes', '1']
+
+    if not force and (switch_time_parsed - now).total_seconds() < 0:
+        return 'Specified switch time lies in the past. Force to set value.', 400
```

Comment:
Do you have a use case for force setting a switch time in the past?

Reply:
the rationale here is to be able to force a node to switch as quickly as possible by setting the switch time to the past